packages/default/scss/dataviz/_variables.scss

```diff
@@ -91,7 +91,7 @@ $kendo-chart-font-family: var( --kendo-font-family, inherit ) !default;
 $kendo-chart-font-size: var( --kendo-font-size, inherit ) !default;
 $kendo-chart-line-height: var( --kendo-line-height, normal ) !default;
 $kendo-chart-tooltip-font-size: calc( #{$kendo-font-size} * .929) !default;
-$kendo-chart-label-font-size: var( --kendo-font-size, inherit ) !default;
+$kendo-chart-label-font-size: var( --kendo-font-size-md, inherit ) !default;
```

Comment:
Prior the previous PR this value was _--kendo-font-size-sm_ - could you please ensure the needed value now is _--kendo-font-size-md_?